### PR TITLE
fix(pacer.utils): add cafc -> 013 mappings

### PR DESCRIPTION
**Pull request by grossir**
Solves #732

### CODE REVIEW

**Review by albertisfu — APPROVED** [review]
This looks good to me!

**Review by mlissner — CHANGES_REQUESTED** [review]
Oh shoot, my comment was pending. There is a change I'd suggest...

File: juriscraper/pacer/utils.py [hunk 1]
```diff
@@ -252,6 +253,7 @@ def get_doc_id_prefix_from_court_id(court_id):
         "cadc": "012",
         "caeb": "032",
         "caed": "033",
+        "cafc": "013",
```
[the remark this answers is not in the document]
Comment (grossir) [reply juriscraper/pacer/utils.py]: Just updated it

**Review by mlissner — APPROVED** [review]

**Comment by mlissner** [comment]
Cool. Merging, thanks!

**Pull request merged**